sys/riscv/conf/std.CHERI-PURECAP

```diff
@@ -0,0 +1,8 @@
+include "std.CHERI"
+
+machine		riscv	riscv64c
+
+makeoptions	NO_MODULES=yes
```

Comment:
Hmm, this feels more like a thing for .mk when `${MACHINE_ABI:Mpurecap}` in general, not an arch-specific kernel config option.

Reply:
Please no, it's a lot easier to work with in my branch where I'm working on kldload if it stays this way for now.

Reply:
Well, no, it doesn't belong in the kernel config, and if you use `?=` in a .mk file then you can continue to set it on the command line whilst only having a single copy of the variable definition shared by all architectures in the "right" place.

Reply:
You can't unset variables on the command line.  And the value doesn't matter, just if it is defined or not (see kern.*.mk where all tests are just `if !defined(NO_MODULES)`.

Reply:
Ugh... I hate that pattern.